packages/api/src/functions/dbAuth/DbAuthHandler.ts

```diff
@@ -52,7 +52,7 @@ interface DbAuthHandlerOptions {
    * Object containing forgot password options
    */
   forgotPassword: {
-    handler: (user: Record<string, unknown>) => Promise<any>
+    handler: (user: TUser) => any
```

Comment:
Have you tried these unknowns in an actual project @Tobbe ? It will degrade the experience to a halt I think - with errors like "obj isn't assignable to type unknown" etc.

This isn't an unknown, its a genuine any. if you look at the description Rob's provided in the auth.ts template.

Reply:
I have made a bunch of these changes locally on your branch already. If you want I can push them, and we can work from there. Or you can do them from scratch on your own 🙂 
Or you don't agree at all, and then we can discuss first 🙂

Reply:
This is a great suggestion! Feel free to push!

Reply:
Skipping this for now. I got side-tracked into types for auth on the web side. I/we can come back to this in another PR